RecoEcal/EgammaClusterAlgos/src/PFECALSuperClusterAlgo.cc

```diff
@@ -145,6 +154,16 @@ void PFECALSuperClusterAlgo::update(const edm::EventSetup& setup) {
   channelStatus_ = esChannelStatusHandle_.product();
 }
 
+void PFECALSuperClusterAlgo::updateSCParams(const edm::EventSetup& setup) {
+  edm::ESHandle<EcalMustacheSCParameters> ecalMustacheSCParamsHandle_ = setup.getHandle(ecalMustacheSCParametersToken_);
+  mustacheSCParamsHelper_ =
+      static_cast<const reco::MustacheSCParametersHelper*>(ecalMustacheSCParamsHandle_.product());
```

Comment:
The ESProduct is `EcalMustacheSCParameters`, it can not be casted to a derived class type.